services/src/main/java/org/keycloak/protocol/oidc/endpoints/LogoutEndpoint.java

```diff
@@ -104,6 +104,7 @@ public Response logout(@QueryParam(OIDCLoginProtocol.REDIRECT_URI_PARAM) String
                            @QueryParam("initiating_idp") String initiatingIdp) {
         String redirect = postLogoutRedirectUri != null ? postLogoutRedirectUri : redirectUri;
 
+        // TODO evaluate importance of removing this: major performance boost observed
```

Comment:
Remove this TODO as it could result in someone removing this by mistake, and doing so would be a breaking change. We are planning on introducing support for RP Initiated logout (https://issues.jboss.org/browse/KEYCLOAK-11603) which won't suffer from the design issue that this logout endpoint does. We will deprecate and eventually remove the legacy KC endpoint.

Reply:
Removed, thanks!